release.js

```diff
@@ -36,15 +36,31 @@ module.exports = async function ({ github, context, inputs }) {
 
   const { opticUrl, npmTag, version, id } = releaseMeta
 
+  const run = runSpawn()
   if (!pr.merged) {
-    return github.rest.repos.deleteRelease({
-      owner,
-      repo,
-      release_id: id,
-    })
+    const branchName = `release/${version}`
+    try {
+      await run('git', ['push', 'origin', '--delete', branchName])
+    } catch (err) {
+      core.setFailed(
```

Comment:
I don't know (and it's not clear from the docs) if calling setFailed also exits the process, let's return just in case it doesn't